src/widget/woverview.cpp

```diff
@@ -1091,8 +1088,8 @@ void WOverview::paintText(const QString& text, QPainter* pPainter) {
 
 double WOverview::samplePositionToSeconds(double sample) {
     // TODO: replace with rate_ratio in PR #1765
-    double rateRatio = 1.0 + m_pRateDirControl->get() * m_pRateRangeControl->get() * m_pRateSliderControl->get();
-    return sample / m_trackSampleRateControl->get() / mixxx::kEngineChannelCount / rateRatio;
+    return sample / m_trackSampleRateControl->get() / 
+            mixxx::kEngineChannelCount / m_pRateRatioControl->get();
```

Comment:
Can't we move this code to a global helper function (maybe `inline`)? I guess there are various places where we need to convert a sample position to seconds.

Also, line 1090  (`// TODO:... `) is obsolete now, so it should be removed. 

Personally, I always find stuff like `a/b/c/d` quite confusing. It's the same as `a/(b*c*d)` right? I prefer that, but that's probably just me.

Reply:
I did not find a similar math in the code. At least I have split the long expression now for a better readability.